src/meta/processors/admin/ClearSpaceProcessor.cpp

```diff
@@ -0,0 +1,103 @@
+/* Copyright (c) 2022 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#include "meta/processors/admin/ClearSpaceProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+void ClearSpaceProcessor::process(const cpp2::ClearSpaceReq& req) {
+  folly::SharedMutex::ReadHolder rHolder(LockUtils::snapshotLock());
+  folly::SharedMutex::WriteHolder holder(LockUtils::lock());
```

Comment:
Why we need to use write lock here? BTW, do we really need to grab the snapshot lock here?